deployment/geonode/templates/geonode/geonode-env.yaml

```diff
@@ -55,6 +55,7 @@ data:
 
   # General settings
   FREETEXT_KEYWORDS_READONLY: {{ include "boolean2str" .Values.geonode.general.freetext_keywords_readonly | quote }}
+  FIXTURE_DIRS: "[ '/usr/src/geonode/fixtures' ]"
```

Comment:
@mwallschlaeger This does not match the actually configured volume:

https://github.com/zalf-rdm/geonode-k8s/pull/50/files#diff-fca9f77d2a6cb907ee65a0f6252b1261757188e043aed583e778fccdc9b1d4baR131

I will have to provide a quick fix for that